spec/services/webhooks/resend_service_spec.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Webhooks::ResendService, type: :service do
+  subject(:resend_service) { described_class.new(webhook:) }
+
+  let(:webhook) { create(:webhook, :failed) }
+
+  it 'enqueues a SendWebhookJob' do
```

Comment:
Maybe another example for ensuring that webhook is correctly assigned to the result response?